src/gui/addnewtorrentdialog.cpp

```diff
@@ -241,6 +243,12 @@ AddNewTorrentDialog::AddNewTorrentDialog(const BitTorrent::AddTorrentParams &inP
 
     connect(m_ui->contentTreeView->header(), &QWidget::customContextMenuRequested, this, &AddNewTorrentDialog::displayColumnHeaderMenu);
 
+    // Torrent content filtering
+    m_filterLine->setPlaceholderText(tr("Filter files..."));
+    m_filterLine->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Fixed);
```

Comment:
Tested, it seems to me that this line and the change in addnewtorrentdialog.ui are redundant. I suppose they can be removed, or do they have some special effects?

Reply:
>or do they have some special effects?

To maximize the size of the lineedit.

Reply:
ok, seems there is no better way to do it.